### PR TITLE
Separate Save and Save As and open file dialogs in better locations.

### PR DESCRIPTION
**Pull request by matt-attack**

### CODE REVIEW

**Comment by matt-attack** [comment]
@pjreed this looks good on my end. Should be good to merge if you get the chance to look it over.

File: mapviz/src/mapviz.ui [hunk 1]
```diff
@@ -350,9 +351,9 @@
     <string>Open Config</string>
    </property>
   </action>
-  <action name="actionSave_config">
+  <action name="actionSaveAs_config">
    <property name="text">
-    <string>Save Config</string>
+    <string>Save Config As</string>
```
Comment (pjreed) [inline mapviz/src/mapviz.ui:356]:
```suggestion
    <string>Save Config As...</string>
```

Common convention is to have menu options that open dialogs end with an ellipsis.

Comment (matt-attack) [reply mapviz/src/mapviz.ui:356]: Somehow I was not aware of that. Interesting

**Comment by pjreed** [comment]
Could you also cherry-pick this into `dashing-devel`?  Thanks!

**Pull request merged**